packages/expo-dev-launcher/expo-dev-launcher.podspec

```diff
@@ -18,6 +18,10 @@ Pod::Spec.new do |s|
   s.requires_arc   = true
   s.header_dir     = 'EXDevLauncher'
 
+  s.resource_bundles = {
+    'EXDevLauncher' => ['ios/assets', 'ios/main.jsbundle']
```

Comment:
I think it'll be better to create a subfolder for those assets and bundle.

Reply:
The way things are set up now the assets will already be in their own subfolder, so this seems fine to me, with no risk of collisions:
<img width="586" alt="Screen Shot 2021-04-22 at 22 34 46" src="https://user-images.githubusercontent.com/497214/115775116-1dda7500-a3bb-11eb-97d6-5bbb4d9593fa.png">

Reply:
Or did you perhaps mean a new subfolder inside the `packages/expo-dev-launcher/ios` subfolder? What should it be named?

Reply:
I meant a new subfolder inside the `packages/expo-dev-launcher/ios` directory. We can call it `resources` for example.